### ISSUE TITLE
Layouts. Add a generic container element.

### ISSUE THREAD

**Issue by pszmyd** [OrchardCMS/Orchard#5334]
Right now we have some "layout" containers that rely on a grid (columns, rows) plus a container that cannot be used directly (canvas). We're missing a generic container (div, section, etc.) that could be placed anywhere and also used in blueprints for customization.

I have such implementation and can push it to the core. @sfmskywalker @DanielStolt, what'd you think?

**Comment by sfmskywalker** [comment]
The Canvas element itself is generic and can be reused and which renders as a div. Would that meet your requirement?

**Comment by pszmyd** [comment]
Yes it would, I guess. But the thing is it's not shown in blueprints (it's not shown at least). So I cannot create specific elements based on this.

**Comment by neTp9c** [comment]
The Layouts module really lacks such simple container elements that will be shown in blueprints.

**Comment by pszmyd** [comment]
I've already done that. It's been a while since latest changes to Layouts module, although I'll try to come up with a pull request. Containers are handled differently than other elements, hence the problem.

**Comment by pszmyd** [comment]
In general - my scenario was to have a generic container (div, section, you name it) that could have some custom classes/IDs on it for the sake of styling, scripting, and such (or making a custom Razor file for it). Something I could drop things in. Is that what you ask for?

**Comment by neTp9c** [comment]
Yes, It's exactly what i am looking for. Can I somehow get this feature?

**Comment by pszmyd** [comment]
It will be sent as a pull request shortly.

**Comment by spero-cnet** [comment]
@pszmyd - great work - this is exactly what I'm looking for - I can't believe a generic container wasn't included as part of the new layout.  How do I implement your solution?

**Comment by neTp9c** [comment]
If you need simple container:
1. In the dev branch of source code the canvas element can be used multiply times (like any other elements). That is rendered as a div.
2. You can download MainBit,Layouts module from gallery (it contains generic container that is rendered as a div).

**Comment by spero-cnet** [comment]
Thanks @neTp9c that's just what I needed.

**Comment by fengweijp** [comment]
@pszmyd that's extactly what I am looking for, where can i somehow get this feature from?
